text/0054-steering-committee-elections.md

```diff
@@ -0,0 +1,158 @@
+
+## Summary
+
+The project governance document will be amended to include a new process for
+electing the membership of the project Steering Committee.
+
+## Motivation
+
+The Paketo Steering Committee is the set of individuals who are responsible and
+accountable for guiding the Paketo project in a holistic manner towards
+achieving its goals. It currently consists of three members who help to
+administer the project and provide feedback and guidance on proposals that have
+project-wide impact.
+
+In the early years of the project, this group of individuals was composed of
+the project founders and then by engineers with significant experience with or
+contributions to the project. The technical process for voting in committee
+members is outlined in the [Paketo Governance
+RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0002-governance.md),
+but it is missing crucial detail on when and how these choices are made.
+
+As the project has evolved to support an ever growing number of users and
+contributors, it has become necessary to reassess this process and propose an
+alternative that will bring in a more diverse set of experiences and
+perspectives.
+
+## Detailed Explanation
+
+### Process Changes
+
+#### Membership
+
+The composition of the Steering Committee will align to the specifications,
+with some modifications, outlined in the [Cloud Foundry Foundation
+Charter](https://github.com/cloudfoundry/community/blob/main/governing-board/charter.md)
+Section 7.b. The modified contents are outlined below.
+
+1. The Steering Committee will be composed of three (3) members.
+2. Each Steering Committee member will be elected to a two-year term.
+3. There will be an annual election to determine the composition of the
+   Steering Committee for the following year. Two seats will be up for election
+   in one year and one will be up for election the following year.
+4. Employees from the same company or Related Companies (as defined in CFF
+   Charter Section 9) should not hold more than one Steering Committee seat.
+   a. During any Steering Committee election, if the Steering Committee
+      membership would exceed this limit even after the natural cycle of Steering
+      Committee seat term expirations, enough Steering Committee members must
+      resign for it to be possible for the election to yield a diverse enough
+      Steering Committee.
+   b. If a Steering Committee election cannot produce a diverse enough Steering
+      Committee, the limit will not apply until the next election cycle. When a
+      change in employment of a Steering Committee member causes the Steering
+      Committee membership to exceed this limit, that Steering Committee member
+      will NOT be required to resign their Steering Committee membership.
+5. One Steering Committee seat is reserved for a “community champion”, defined
+   as a person NOT working for a corporation that is a paying member of the
+   Cloud Foundry Foundation.
```

Comment:
And would 4.b also apply to this requirement? If no one from the community steps forward, then we'd waive this requirement until the next election.

Reply:
Yes. We would waive the requirement. I can add a clause stating that.

Reply:
Done in the latest version.